src/env.js

```diff
@@ -37,3 +45,13 @@ export default env;
 export function isMac( userAgent ) {
 	return userAgent.indexOf( 'macintosh' ) > -1;
 }
+
+/**
+ * Checks if User Agent represented by the string is Edge browser.
+ *
+ * @param {String} userAgent **Lowercase** `navigator.userAgent` string.
+ * @returns {Boolean} Whether User Agent is Edge or not.
+ */
+export function isEdge( userAgent ) {
+	return userAgent.indexOf( 'edge' ) > -1;
```

Comment:
That's unsafe. There are many words with "edge" in them. Something more precise would be better.

Reply:
BTW, feel free to look into CKE4's code:

https://github.com/ckeditor/ckeditor-dev/blob/master/core/env.js#L20

Reply:
@Reinmar I'm not 100% sure but it looks like `[ /]` in that regex could be restricted to:

```js
/edge\/(\d+.?\d*)/
```

I couldn't find why there is space allowed...

Reply:
ps.: https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/hh869301(v=vs.85)

Describes only `Edge/<EdgeHTML Rev>.<Windows Build>` as possible value so I've set the regex accordingly to this.